demo/bulk_insert/bulk_insert.py

```diff
@@ -151,27 +179,33 @@ def ProcessNodes(nodes_csv_files):
 			depleted_labels += 1
 	# Insert all remaining nodes
 	QueryRedis(labels, NODES)
```

Comment:
Previous calls to QueryRedis followed by some cleanup/update logic, shouldn't we perform the same here?